tools/test-proxy/docker/dockerfile

```
@@ -0,0 +1,42 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0-focal AS build
```

Two questions about the docker image tag:

1. Do we need to use Ubuntu instead of Debian?  If Debian works, I would drop the `-focal`
2. I recommend specifying the full version for more repeatable builds.  So `5.0.301` instead of `5.0`

```suggestion
FROM mcr.microsoft.com/dotnet/sdk:5.0.301 AS build
```